testing/sdk_install.py

```diff
@@ -221,6 +208,37 @@ def uninstall(
             shakedown.pretty_duration(finish - cleanup_start),
             shakedown.pretty_duration(finish - start)))
 
+    state_summary = sdk_cmd.cluster_request('GET', '/mesos/state-summary').json()
+
+    # Sanity check: There should be no dangling resources in the state summary (DCOS-30314)
+    found_dangling_resources = False
+    service_role = '{}-role'.format(service_name)
+    for agent in state_summary['slaves']:
+        # resources should be grouped by role. check for any resources in our expected role:
+        matching_reserved_resources = agent['reserved_resources'].get(service_role, {})
+        if len(matching_reserved_resources) > 0:
```

Comment:
We could just check `if matching_reserved_resources:` (it's possibly a little more pythonic). Then, we could also change the line above to `matching_reserved_resources = agent['reserved_resources'].get(service_role)` which may be a small optimization.

Reply:
Done